icons/qbittorrent.svg

```diff
@@ -0,0 +1 @@
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>qbittorrent</title><path d="M11.7483.0029c-3.7762.08-7.5141 1.8234-9.6447 5.1927C-2.447 11.3938.7663 21.052 8.12 23.2974c7.4518 2.8498 16.1641-3.376 15.8719-11.348A11.9977 11.9977 0 0 0 21.615 4.794C19.3445 1.5096 15.5289-.0772 11.7483.0029Zm.0375 1.7158c3.262-.0691 6.5517 1.2659 8.4603 4.0363l.0094.0125.0094.014a10.3256 10.3256 0 0 1 2.0549 6.1804V11.9947c.2585 6.7215-7.35 12.156-13.628 9.7338l-.0297-.011-.0296-.0093C2.421 19.8326-.3911 11.379 3.4663 6.1613l.0187-.025.0156-.0266c1.7896-2.8481 5.0233-4.322 8.2853-4.391Zm1.3533 2.0486v12.8998h1.3471v-1.1705c.5748.9206 1.3663 1.3814 2.3768 1.3814 1.0153 0 1.8869-.4375 2.6127-1.311.7251-.8743 1.0876-2.1114 1.0876-3.713 0-.6745-.0859-1.308-.2578-1.9001-.172-.5922-.411-1.0979-.7173-1.5142a3.0122 3.0113 0 0 0-1.161-.9767c-.4677-.2341-.9647-.3516-1.4908-.3516-.9513 0-1.7332.4187-2.3455 1.258v-4.602ZM7.062 7.1114a3.2568 3.2558 0 0 0-1.8705.5798c-.5643.382-.9964.9443-1.297 1.6892-.2957.7457-.4438 1.587-.4438 2.5252 0 1.5434.3647 2.7578 1.0955 3.6442.7363.8854 1.613 1.3298 2.6284 1.3298a2.571 2.5702 0 0 0 1.2891-.3438c.4085-.2294.7262-.5166.9517-.8626v4.5755h1.4501V7.3224H9.5606v1.2485c-.6076-.9734-1.441-1.4595-2.4987-1.4595ZM7.1995 8.36c.6339 0 1.184.3148 1.6517.9423.4725.628.7094 1.5622.7094 2.8065 0 1.1667-.2227 2.0389-.6688 2.6143-.446.569-.9775.8532-1.5955.8532-.6458 0-1.1982-.2992-1.661-.8985-.4614-.6034-.6939-1.5077-.6939-2.7097 0-1.237.2183-2.1473.6532-2.7284.4404-.5865.9758-.8797 1.6049-.8797zm9.6072.0531c.629 0 1.1644.2917 1.6048.872.4453.581.6688 1.4807.6688 2.7002 0 1.1908-.2317 2.0898-.6938 2.694-.462.5978-1.01.897-1.6439.897-.7794 0-1.3968-.4083-1.8533-1.2235-.2797-.4995-.4188-1.308-.4188-2.4284h-.0016c0-1.1388.2318-2.0055.6938-2.605.463-.6041 1.01-.9063 1.644-.9063z"/></svg>
```

Comment:
```suggestion
<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>qbittorrent</title><path d="M21.624 4.794C19.5 1.712 15.82-.076 11.742.008c-4.065.086-7.68 2.03-9.636 5.147C-.038 8.055-.584 12 .647 15.71c1.233 3.714 4.034 6.554 7.433 7.578 1.226.473 2.522.706 3.83.706 2.42 0 4.884-.795 7.031-2.329 3.302-2.36 5.19-5.994 5.052-9.728a12.06 12.06 0 0 0-2.37-7.144zm-3.656 15.51c-2.848 2.036-6.32 2.569-9.345 1.403-2.938-.888-5.325-3.326-6.386-6.524-1.06-3.193-.606-6.57 1.25-9.085 1.692-2.691 4.79-4.342 8.29-4.416l.244-.002c3.402 0 6.464 1.508 8.242 4.09a10.397 10.397 0 0 1 2.055 6.184l.001.032c.122 3.174-1.504 6.284-4.351 8.318zM19.585 8.43a3.014 3.014 0 0 0-1.161-.977 3.289 3.289 0 0 0-1.492-.352c-.952 0-1.734.419-2.347 1.259V3.755h-1.452v12.908h1.347v-1.17c.575.92 1.368 1.381 2.38 1.381 1.015 0 1.886-.437 2.613-1.311.725-.875 1.088-2.114 1.088-3.717 0-.675-.086-1.309-.258-1.901s-.411-1.098-.718-1.515zm-1.201 6.243c-.462.599-1.011.899-1.646.899-.78 0-1.398-.408-1.855-1.224-.28-.5-.419-1.31-.419-2.431 0-1.139.23-2.007.693-2.607.463-.604 1.011-.906 1.645-.906.63 0 1.165.29 1.606.871.446.582.67 1.483.67 2.703 0 1.192-.232 2.09-.694 2.695zM9.552 8.562c-.608-.974-1.442-1.461-2.5-1.461a3.26 3.26 0 0 0-1.872.58c-.565.383-.998.946-1.299 1.691-.296.746-.444 1.588-.444 2.527 0 1.544.366 2.76 1.097 3.646.737.886 1.614 1.33 2.63 1.33.453.001.898-.118 1.29-.344.409-.23.726-.516.952-.863v4.579h1.452V7.312H9.552v1.25zm-.67 6.155c-.446.57-.978.855-1.597.855-.646 0-1.199-.3-1.662-.9-.462-.603-.694-1.508-.694-2.71v-.001c0-1.238.219-2.148.654-2.73.44-.587.976-.88 1.605-.88.635 0 1.186.314 1.654.942.473.628.71 1.564.71 2.81 0 1.167-.224 2.039-.67 2.614z"/></svg>
```

This might fix the issue.

Reply:
@simple-icons/icon-review Can someone take a look at this since I suggested changes.